server/handlers/user_handler.go

```diff
@@ -120,3 +120,14 @@ func (h *Handler) UserPrefsHandler(w http.ResponseWriter, req *http.Request, pre
 		return
 	}
 }
+
+func (h *Handler) ShareDesignHandler(w http.ResponseWriter, r *http.Request, prefObj *models.Preference, user *models.User, provider models.Provider) {
+	statusCode, err := provider.ShareDesign(r)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("Error: %v", err.Error()), statusCode)
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	fmt.Fprint(w, "Design shared successfully")
```

Comment:
No more “successfully”s, please. 
```suggestion
	fmt.Fprint(w, "Design shared")
```